test.js

```diff
@@ -86,7 +86,8 @@ global.Runners = {
       iris300: createForkObject('iris', 700),
       hop400: createForkObject('hop', 1000),
       hop401: createForkObject('hop401', 1010),
-      fingerroot500: createForkObject('fingerroot', 1350)
+      fingerroot500: createForkObject('fingerroot', 1350),
+      tbd600: createForkObject('tbd600', 2000)
```

Comment:
Updated! The rsk blockchain height right before this fork is activated is about 488. I think it would be enough with 1600 for the activation of this fork.